DigiMeSDK/Core/Classes/Security/DMECrypto.h

```diff
@@ -52,6 +52,42 @@ NS_ASSUME_NONNULL_BEGIN
  */
 - (nullable NSData *)decryptAes256UsingKey:(NSData *)keyData initializationVector:(NSData *)ivData data:(NSData *)data error:(NSError * __autoreleasing *)error;
 
+/**
+ Generates random data using length as a parameter.
+ 
+ @param length int
+ @return NSData - rundom bytes for the specified length.
+ */
+- (NSData *)getRandomUnsignedCharacters:(int)length;
+
+/**
+ Encrypt metadata for Postbox with AES encryption
+ 
+ @param symmetricalKey NSData
+ @param iv NSData
+ @param metadata NSData
+ @return NSString - AES encrypted metadata to push to postbox in Base64 encoding.
+ */
+- (NSString *)preparePostboxMetadataWithKey:(NSData *)symmetricalKey initializationVector:(NSData *)iv metadata:(NSData *)metadata;
```

Comment:
The description doesn't quite match the method name. Perhaps a more accurate method name would be

`encryptMetadata: <metadata_here> symmetricalKey: <symmetricalKey_here> initializationVector: <iv_here>`